chain/network/src/config_json.rs

```diff
@@ -161,9 +177,8 @@ pub struct Config {
     /// Used only if this node is a validator and public_ips is empty (see
     /// description of public_ips field).  Format `<domain/ip>:<port>`, for
     /// example `stun.l.google.com:19302`.
-    // TODO: unskip, once the functionality is implemented.
-    #[serde(skip)] // TODO: add a default list.
-    pub trusted_stun_servers: Vec<String>,
+    #[serde(default = "default_trusted_stun_servers")]
+    pub trusted_stun_servers: Vec<stun::ServerAddr>,
```

Comment:
I'd also mention that we'll query all of them in parallel (and not proceed if any of them returns a different ip than the rest)